modules/features/manatime_clients/manatime_clients.features.field_instance.inc

```diff
@@ -0,0 +1,109 @@
+<?php
```

Comment:
Right now both features are duplicated. The original version (without `feature` in the name) should be deleted.